std-bits/bench-processor/src/main/java/org/enso/benchmarks/processor/BenchProcessor.java

```diff
@@ -0,0 +1,233 @@
+    } catch (IOException e) {
+      System.err.println("Failed to generate source file for group '" + group.name() + "': " + e.getMessage());
+    }
+  }
+
+  private void generateClassForGroup(Writer javaSrcFileWriter, String moduleName, BenchGroup group) throws IOException {
+    String groupFullClassName = createGroupClassName(group);
+    System.out.println("[mylog] Generating spec code for group '" + groupFullClassName + "'");
+    String className = groupFullClassName.substring(groupFullClassName.lastIndexOf('.') + 1);
+    List<BenchSpec> specs = group.specs();
+    List<String> specJavaNames = specs
+        .stream()
+        .map(spec -> normalize(spec.name()))
+        .collect(Collectors.toUnmodifiableList());
+
+    javaSrcFileWriter.append("package " + generatedSourcesPackagePrefix + ";\n");
+    javaSrcFileWriter.append("\n");
+    javaSrcFileWriter.append(String.join("\n", imports));
+    javaSrcFileWriter.append("\n");
+    javaSrcFileWriter.append("\n");
+    javaSrcFileWriter.append("/**\n");
+    javaSrcFileWriter.append(" * Generated from:\n");
+    javaSrcFileWriter.append(" * - Module: " + moduleName + "\n");
+    javaSrcFileWriter.append(" * - Group: \"" + group.name() + "\"\n");
+    javaSrcFileWriter.append(" * Generated by {@link " + getClass().getName() + "}.\n");
+    javaSrcFileWriter.append(" */\n");
+    javaSrcFileWriter.append("@BenchmarkMode(Mode.AverageTime)\n");
+    javaSrcFileWriter.append("@Fork(1)\n");
+    javaSrcFileWriter.append("@State(Scope.Benchmark)\n");
+    javaSrcFileWriter.append("public class " + className + " {\n");
+    javaSrcFileWriter.append("  private Value groupInputArg;\n");
+    for (var specJavaName : specJavaNames) {
+      javaSrcFileWriter.append("  private Value benchFunc_" + specJavaName + ";\n");
+    }
+    javaSrcFileWriter.append("  \n");
+    javaSrcFileWriter.append("  @Setup\n");
+    javaSrcFileWriter.append("  public void setup(BenchmarkParams params) throws Exception {\n");
+    javaSrcFileWriter.append("    var ctx = Context.newBuilder()\n");
+    javaSrcFileWriter.append("      .allowExperimentalOptions(true)\n");
+    javaSrcFileWriter.append("      .allowIO(IOAccess.ALL)\n");
+    javaSrcFileWriter.append("      .allowAllAccess(true)\n");
+    javaSrcFileWriter.append("      .logHandler(new ByteArrayOutputStream())\n");
+    javaSrcFileWriter.append("      .option(\n");
+    javaSrcFileWriter.append("        \"enso.languageHomeOverride\",\n");
+    javaSrcFileWriter.append("        Paths.get(\"../../distribution/component\").toFile().getAbsolutePath()\n");
+    javaSrcFileWriter.append("      ).build();\n");
+    javaSrcFileWriter.append("    \n");
+    javaSrcFileWriter.append("    Value module = ctx.getBindings(LanguageInfo.ID).invokeMember(MethodNames.TopScope.GET_MODULE, \"" + moduleName + "\");\n");
+    javaSrcFileWriter.append("    File benchProjectDir = new File(\"" + benchRootDir.getAbsolutePath() + "\");\n");
```

Comment:
The generated source code contains fully qualified name:
```java
    File benchProjectDir = new File("/home/enso/enso/test/Benchmarks");
    File languageHomeOverride = new File("/home/enso/enso/distribution/component");
```
that means the generated code is different on each computer. It'd be better if the generated code was always the same.

Please replace with relative lookup with the help of `ProtectionDomain.getCodeSource().getLocation()`.

Reply:
Done in f5622840c9ec7e3a4f978a57898f21b238f88945